rules/integrations/cloud_defend/container_workload_protection.toml

```diff
@@ -0,0 +1,32 @@
+[metadata]
+creation_date = "2023/03/29"
+integration = ["cloud_defend"]
+maturity = "production"
+min_stack_comments = "Initial version of cloud_defend alerts rule"
+min_stack_version = "8.8.0"
+updated_date = "2023/03/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time a 'Container Workload Protection' alert is received. Enabling this rule allows you to
+immediately begin triaging and investigating these alerts.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-cloud_defend.alerts-*"]
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Container Workload Protection"
+risk_score = 47
+rule_id = "4b4e9c99-27ea-4621-95c8-82341bc6e512"
+rule_name_override = "message"
+severity = "medium"
+tags = ["Elastic", "Container Workload Protection"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.kind:alert
```

Comment:
Will the `event` object be populated with a `module`? That is the most consistent way to exclude this rule from our generic external event promotion [rule](https://github.com/elastic/detection-rules/blob/main/rules/promotions/external_alerts.toml), where we would do

```sql
event.kind:alert and not event.module:(endgame or endpoint or container)
```

Otherwise, this will _also_ trigger  that rule every time.

Reply:
I see. That could be added easily. We could set     event.module: cloud_defend   if that works.

Reply:
Updated with event.module on both rules.